<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-SIMJPDD" failures="0" tests="8" name="testCases.url1TestCases" time="36.559" errors="1" timestamp="2021-11-03T19:17:43 IST" skipped="0">
  <testcase classname="testCases.url1TestCases" name="staticDropDownCase1" time="2.917"/>
  <system-out/>
  <testcase classname="testCases.url1TestCases" name="staticDropDownCase2" time="2.310"/>
  <system-out/>
  <testcase classname="testCases.url1TestCases" name="staticDropDownCase8" time="16.266"/>
  <system-out/>
  <testcase classname="testCases.url1TestCases" name="staticDropDownCase7" time="1.093"/>
  <system-out/>
  <testcase classname="testCases.url1TestCases" name="staticDropDownCase4" time="5.549"/>
  <system-out/>
  <testcase classname="testCases.url1TestCases" name="staticDropDownCase3" time="6.296"/>
  <system-out/>
  <testcase classname="testCases.url1TestCases" name="staticDropDownCase6" time="1.022"/>
  <system-out/>
  <testcase classname="testCases.url1TestCases" name="staticDropDownCase5" time="1.106">
    <error message="expected:&lt;&gt; but was:&lt;Seachhhhasd&gt;" type="junit.framework.ComparisonFailure">
      <![CDATA[junit.framework.ComparisonFailure: expected:<> but was:<Seachhhhasd>
at junit.framework.Assert.assertEquals(Assert.java:81)
at junit.framework.Assert.assertEquals(Assert.java:87)
at testCases.url1TestCases.staticDropDownCase5(url1TestCases.java:134)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:78)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- staticDropDownCase5 -->
  <system-out/>
</testsuite> <!-- testCases.url1TestCases -->
